src/OnlineRetailPortal.Contracts/Models/Service/UpdateProductEntity.cs

```diff
@@ -7,15 +7,14 @@ namespace OnlineRetailPortal.Contracts
     public class UpdateProductEntity
     {
         public string Id { get; set; }
-        public string SellerId { get; set; }
         public string Name { get; set; }
         public string Description { get; set; }
         public string HeroImage { get; set; }
         public Price Price { get; set; }
-        public Category Category { get; set; }
-        public Status Status { get; set; }
-        public DateTime PostDateTime { get; set; }
-        public DateTime ExpirationDate { get; set; }
+        public string Category { get; set; }
+        public string Status { get; set; }
```

Comment:
why is Status of type string when you have Status class which contains (Active,Disabled etc)?

Reply:
Its deleted we are not uisng it